### PR TITLE
Add DeployerEnvironmentInfo info in TaskLauncher SPI

### LINKED ISSUE #204: Add platform info in TaskLauncher

### PR DESCRIPTION
**Pull request by trisberg**
Resolves #204

### CODE REVIEW

**Comment by ericbottard** [comment]
Should we rename to `RuntimeEnvironmentInfo` or something similar? (and use "runtime" in place of deployer in every touched comment)

**Comment by trisberg** [comment]
we can do that, let me push new change

File: spring-cloud-deployer-spi/src/main/java/org/springframework/cloud/deployer/spi/app/AppDeployer.java [hunk 1]
```diff
@@ -139,11 +138,9 @@
 	AppStatus status(String id);
 
 	/**
-	 * Return the environment info for this deployer. This info should include the
-	 * following: 'deployer-class-name','deployer-version', 'deployer-spi-version', 'platform-type',
-	 * 'platform-client-version', 'platform-host-version', 'java-version', 'spring-version', 'spring-boot-version'.
+	 * Return the environment info for this deployer.
 	 *
-	 * @return the app deployment environment info
+	 * @return the deployer environment info
```
Comment (ericbottard) [inline spring-cloud-deployer-spi/src/main/java/org/springframework/cloud/deployer/spi/app/AppDeployer.java]: should read "runtime", can change on merge

**Comment by trisberg** [comment]
Merged via https://github.com/spring-cloud/spring-cloud-deployer/commit/a38c4dce26b95313c65f71ae88d78152ede1cf5a

**Pull request closed without merging**